app/src/androidTest/java/net/skyscanner/backpack/compose/BpkSnapshotTest.kt

```diff
@@ -91,11 +89,7 @@ open class BpkSnapshotTest(private val tags: List<Any> = emptyList()) : Screensh
         scenario.onActivity { activity ->
             activity.setContent {
                 @Suppress("DEPRECATION")
-                activity.window.clearFlags(
-                    FLAG_TRANSLUCENT_STATUS or
-                        SYSTEM_UI_FLAG_LAYOUT_FULLSCREEN or
-                        SYSTEM_UI_FLAG_LAYOUT_STABLE,
-                )
+                activity.window.addFlags(SYSTEM_UI_FLAG_LAYOUT_FULLSCREEN)
```

Comment:
with the bottom sheet the modal takes up the whole screen, so we need to make sure we don't have a nav bar as otherwise the snapshot will change everytime (as the time will be different). this will probably affect a few other tests that are at full height